analyzer/src/main/kotlin/PackageCurationProvider.kt

```diff
@@ -32,7 +32,7 @@ import org.ossreviewtoolkit.utils.common.getDuplicates
 interface PackageCurationProviderFactory<CONFIG> : ConfigurablePluginFactory<PackageCurationProvider> {
     companion object {
         val ALL = Plugin.getAll<PackageCurationProviderFactory<*>>()
-        const val ORT_YML_PROVIDER_ID = "ort-yml"
+        const val ORT_YML_PROVIDER_ID = "OrtYml"
```

Comment:
As discussed, probably also rename this to `RepositoryConfiguration` to make more clear where the `.ort.yml` comes from (and that it actually also could be a JSON).